sources/packages/backend/libs/sims-db/src/entities/cas-supplier.model.ts

```diff
@@ -148,3 +159,16 @@ export interface SupplierAddress {
   siteProtected?: string;
   lastUpdated: Date;
 }
+/**
+ * Student profile snapshot information.
+ */
+interface StudentProfileSnapshot {
+  givenName: string;
```

Comment:
Since this is the snapshot of the student profile I would recommend that it uses the "SIMS tables" conventions where `givenName` would be `firstName`. Not a blocker though.